app/controllers/api/vcs.php

```diff
@@ -830,7 +965,7 @@
         $repository['organization'] = $installation->getAttribute('organization', '');
         $repository['provider'] = $installation->getAttribute('provider', '');
 
-        $response->dynamic(new Document($repository), Response::MODEL_PROVIDER_REPOSITORY);
+        $response->dynamic(new Document($repository), Response::MODEL_RUNTIME_PROVIDER_REPOSITORY);
```

Comment:
Let's have generic provider repository (without runtime or framework) for this scenario, since we are not doing any detection.

This is nice follow-up to my first comments when I suggsted abstract base class to extend - this is good reas to make it non-abstract and actually use it in responses